charts/iter8/templates/_payload-slack.tpl

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Payload templates can be hosted anywhere. Perhaps we should simply create a gist for the (sample slack/GitHub) payload templates.

Reply:
Okay, sounds good to me.